### ISSUE TITLE
M1: `pip install proposal` on MacOS only works with manual `pip install cmake`

### ISSUE THREAD

**Issue by Jean1995** [tudo-astroparticlephysics/PROPOSAL#340]
After conan solved the issue with boost on M1 (see [here](https://github.com/conan-io/conan-center-index/issues/14869)), `pip install proposal` should basically work now.

However, `pip install proposal` still fails with an error from pip:

``` 
      error: You need cmake >= 3.16
```

If I maunally do `pip install cmake`, and try `pip install proposal` again, then this error doesn't appear anymore.
I thought that pip would automatically solve the `cmake` depencency?

**Comment by maxnoe** [comment]
Yes it should. Does the pypi distribution for some reason not contain pyproject.toml? Or is the pip Version ancient?

**Comment by maxnoe** [comment]
Seems to be this issue: https://github.com/scikit-build/cmake-python-distributions/issues/324
